components/crud-web-apps/tensorboards/backend/app/utils.py

```diff
@@ -0,0 +1,39 @@
+from kubeflow.kubeflow.crud_backend import helpers
+
+
+def parse_tensorboard(tensorboard):
+    """
+    Process the Tensorboard object and format it as the UI expects it.
+    """
+
+    parsed_tensorboard = {
+        "name": tensorboard['metadata']['name'],
+        "namespace": tensorboard['metadata']['namespace'],
+        "logspath": tensorboard['spec']['logspath'],
+        "age": {
```

Comment:
Could you add a `get_age(k8s_object)` in our common [helpers.py](https://github.com/kubeflow/kubeflow/blob/e99b5e18697a15088abea12543bd4e3f180ff984/components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/helpers.py) file that would return a dictionary with the `uptime` and `timestamp` values? Since other web apps will want to use this convention, for returning age information to their frontends, it would be a good idea to add this logic into a common place.

For now it could only work with dict objects, like CRs, and later one we could extend it to work with class objects as well, such as PVCs for example.

Reply:
Thanks for the comment. I totally agree, so I just added a new commit that adds `get_age(k8s_object)` to the common code.